scripts/upgrade

```diff
@@ -63,74 +67,63 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
-#=================================================
-# CHECK THE PATH
-#=================================================
-
-# Normalize the URL path syntax
-path_url=$(ynh_normalize_url_path "$path_url")
-
 #=================================================
 # STANDARD UPGRADE STEPS
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-
 ynh_script_progression --message="Installing app dependencies"
-ynh_install_app_dependencies $pkg_dependencies
 
+ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
 
 # Now that postgresql is installed, check the db exists and the user is set up
 ynh_script_progression --message="Setting up database"
 if [ -z "$db_pwd" ]; then
 	ynh_psql_test_if_first_run
-	ynh_psql_setup_db "$db_user" "$db_name"
-	db_pwd=$(ynh_app_setting_get "$app" psqlpwd)
+	ynh_app_setting_set --app=$app --key=db_name --value=$db_name
```

Comment:
Why did you remove the `ynh_psql_setup_db`? Previous versions did not have a postgresql database and this is needed on ugprade.